CHANGELOG.md

```diff
@@ -23,9 +23,7 @@ and the project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.
 ### Changed
 
 - Select search hyperparameters depending on which algorithm and single-step model are used ([#30](https://github.com/microsoft/syntheseus/pull/30)) ([@kmaziarz])
-- Add option to override time tolerance in algorithm tests ([#25](https://github.com/microsoft/syntheseus/pull/25)) ([@austint])
```

Comment:
Why is this line deleted? I don't think this was removed.

Reply:
(As discussed offline, I sneaked in a slight cleanup of unimportant CHANGELOG entries)